README.md

```diff
@@ -1,30 +0,0 @@
-This package provides a simple abstraction to an object store for when
-you don't care about keys in the store.
-
-At the moment that object store is S3, and the URL must be provided by
-the environment variable `BLOBS_S3_URL`. The store name should be provided
```

Comment:
Providing environment variable config is *good*, however these ought to also be configurable via the client API.

Reply:
Actually the README was wrong and they were only configurable via the API! I am adding the environment variables back, modified from their previous iteration.